blueprint/index.md

```diff
@@ -1,102 +1,114 @@
+3. Create an access key for the IAM user. For more information, see [Managing access keys for IAM users](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_credentials_access-keys.html "Opens Managing access keys for IAM users") in the AWS documentation.
+4. Write down the access key and secret.
+5. Create an .env file in the directory folder and provide values for the following variables: `AWS_REGION`, `AWS_ACCESS_KEY_ID`, and `AWS_SECRET_ACCESS_KEY`.
+
+   :::primary
+   **Tip**: Start with the sample.env file for this blueprint and provide your org-specific details.
+   :::
 
-### Install and activate the Chat Translator integration app on Genesys Cloud
+### Install and activate the Chat Translator in Genesys Cloud
+
+1. Log in to your Genesys Cloud organization and add an integration. For more information, see [Add an integration](https://help.mypurecloud.com/?p=135807 "Opens the Add an integration article") in the Genesys Cloud Resource Center.
+
+   ![Add integration](images/add-integration.png "Add integration")
+
+2. Install the **Interaction Widget** integration. For more information, see [Set up an Interaction Widget integration](https://help.mypurecloud.com/?p=229319 "Opens the Set up an Interaction Widget integration article") in the Genesys Cloud Resource Center.
 
-1. Login to your Genesys Cloud organization and add a new **Integration**.
-   ![Add Integration](images/add-integration.png "Add Integration")
-   
-2. Install the **Interaction Widget**.
    ![Install Interaction Widget](images/install-interaction-widget.png "Install Interaction Widget")
 
-3. **(Optional)** Use the Name box to give the app a different name.
-   ![Name Interaction Widget](images/name-interaction.png "Name Interaction Widget")
-   
-4. Click the **Configuration** tab. In the **Application URL** box, type the URL of the web application. Be sure to specify the full URL, including the https: at the beginning of the URL.
-   
-   The pcConversationId URL parameter is used to determine the conversation interaction and pcLangTag is used to determine the agent's language in the Chat Translator solution.
+3. (Optional) Use the **Name** box to give the widget a meaningful name. For example, **Chat Translator**.
+
+   ![Interaction Widget name](images/name-interaction.png "Interaction Widget name")
 
-   <pre class="language-nohighlight"><code class="language-nohighlight">https://localhost/?conversationid=&#123;&#123;pcConversationId&#125&#125;&language=&#123;&#123;pcLangTag&#125&#125;</code></pre>
+4. Click the **Configuration** tab.
+5. In the **Application URL** box, type the URL of the web application. Be sure to specify the full URL, beginning with `https:`.
 
-   **(Optional)** Select a group for filtering user acces to the widget.
+   ```
+   https://localhost/?conversationid={{pcConversationId}}&language={{pcLangTag}}
```

Comment:
We will need to use the original format to show this line of code in the Developer Center. Having a {{ }} in our markdown will cause issues in the site itself and will not be able to render it properly.